compiler/qsc_linter/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "qsc_linter"
+
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+homepage.workspace = true
+license.workspace = true
+repository.workspace = true
+
+[dependencies]
+miette = { workspace = true }
+qsc_ast = { path = "../qsc_ast" }
+qsc_fir = { path = "../qsc_fir" }
```

Comment:
Can the fir dependency be removed? I didn't see the fir being used, but maybe I missed it.

Reply:
Nice catch